### PR TITLE
RHIROS-1117 - Refactor processors

### PR DESCRIPTION
**Pull request by upadhyeammit**
## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation update? :memo:

- [ ] Yes
- [ ] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [x] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: ros/processor/inventory_events_consumer.py [hunk 1]
```diff
@@ -1,98 +1,62 @@
...
+                handler = self.event_type_map[event_type]
+                self.try_to_handle_msg(handler(decoded_msg), org_id, processing_err)
+            else:
+                self.LOG.info(
+                    f"{self.prefix} - Event Handling is not found for event {event_type}"
                 )
-            finally:
-                self.consumer.commit()
-        LOG.warning("Stopping inventory consumer")
-        self.consumer.close()
```
Comment (kgaikwad) [inline ros/processor/inventory_events_consumer.py:88]: Hello @patilsuraj767  - 
In inventory related processor, we are explicitly closing the consumer whereas in engine related processor, we are not closing it. 
Could you share few inputs & any scenario you can think of might cause issue when we close on engine side too?

Comment (patilsuraj767) [reply ros/processor/inventory_events_consumer.py:88]: closing the consumer connection is good practice but I guess not mandatory. Basically as per me closing the connection just speed up the process. Example - Consider there is a pod consuming msgs from Kafka but suddenly pod dies for some reason without closing the connection and then OpenShift spin up the new pod but new pod will not be quickly able to connect to Kafka topic till the timeout (30s default) happens and Kafka broker terminates the previous connection.

Comment (upadhyeammit) [reply ros/processor/inventory_events_consumer.py:88]: This makes it better then! I would close the connections for both! Thanks for inputs!

**Comment by upadhyeammit** [comment]
I have rebased this! request to review!

**Comment by upadhyeammit** [comment]
I am closing this pull request now, there is rewrite planned and it will help to resolve this kind of changes

**Pull request closed without merging**